runHarness/WorkloadDrivers/AuctionKubernetesWorkloadDriver.pm

```diff
@@ -35,11 +35,13 @@ has 'namespace' => (
 );
 
 override 'initialize' => sub {
-	my ( $self, $paramHashRef ) = @_;
+	my ( $self, $isMaster ) = @_;
 	super();
 
 	my $cluster = $self->host;
-	$self->namespace($cluster->kubernetesGetNamespace($self->workload->instanceNum, ""));
+	if ($isMaster) {
+		$self->namespace($cluster->kubernetesGetNamespace($self->workload->instanceNum, ""));
+	}
```

Comment:
Will the secondaries work with an empty string for namespace?  Or is it set somewhere else?

Reply:
I added a line to set the namespace of a secondary driver to that of the master